tests/macros/thasargofname.nim

```diff
@@ -0,0 +1,20 @@
+discard """
```

Comment:
not needed if you follow https://github.com/nim-lang/Nim/pull/16233/files#r534416096